app/views/projects/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Projects</h1>
+
+<% @project.each do |project| %>
```

Comment:
Overall this set up is good, but you _should_ only have one instance of `@project` here, and we wouldn't need to loop through it using `.each`. Otherwise, your page is set up correctly.